app/src/gui/components/notebook/datagrid_toolbar.tsx

```diff
@@ -54,33 +64,116 @@ export function GridToolbarSearchRecordDataButton(props: ToolbarProps) {
 
   return (
     <Box style={{marginTop: '4px'}} mr={1}>
-      <Grid container spacing={1}>
-        <Grid item alignItems="stretch" style={{display: 'flex'}}>
-          <Button
-            variant="text"
-            size={'small'}
-            onClick={handleClear}
-            data-testid="searchReset"
-          >
-            reset
-          </Button>
+      <Grid container spacing={1} alignItems="center">
+        {/* Search Bar */}
+        <Grid item xs={9} sm={10} md={11}>
           <TextField
-            label="Search record data (case sensitive)"
+            placeholder="Search for records address"
```

Comment:
Search is not just over the address field and if you want more specific text here it needs to come from the notebook uiSpec and there needs to be an option to update it in the designer.